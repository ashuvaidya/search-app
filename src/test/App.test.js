import React from "react";
import { render } from "@testing-library/react";
import App from "../App.js";

describe("testing for App", () => {
  it("snapshot for App", () => {
    const { container } = render(<App />);
    expect(container).toMatchSnapshot();
  });
});
