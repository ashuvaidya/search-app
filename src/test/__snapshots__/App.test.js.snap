// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing for App snapshot for App 1`] = `
<div>
  <div
    class="App"
  >
    <div
      class="header"
    >
      <h1>
        Search Engine
      </h1>
    </div>
    <div
      class="container"
    >
      <div
        class="search-section"
      >
        <input
          aria-label="search"
          class="search-box"
          data-testid="input-search"
          type="search"
          value=""
        />
        <button
          class="search-button"
          data-testid="searchBtn"
          name="searchBtn"
        >
          Search
        </button>
      </div>
      <div
        class="result-section"
      />
    </div>
  </div>
</div>
`;
